src/toasts/toasts.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const toastsContainer = document.createElement("div");
+document.body.appendChild(toastsContainer);
+
+export function showToast(options) {
```

Comment:
@Andu033 @joeldenning is it possible to use jsdoc/types here in order to know the signature of the options being expected?

Reply:
We could add typescript to esm-styleguide. I think it's okay to start without typescript and add it in as a future PR, if we want it.